### PR TITLE
3 list multiple spaces

### PR DESCRIPTION
**Pull request by GiadaSimonetti**
#3 closed

### CODE REVIEW

File: spec/helpers.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+                    from_date: '12/10/2018',
+                    to_date: '19/10/2018')
+
+  visit('/spaces/new')
+  fill_in :name, with: name
+  fill_in :description, with: description
+  fill_in :price, with: price
+  fill_in :from_date, with: from_date
+  fill_in :to_date, with: to_date
+end
```
Comment (Tagrand) [inline spec/helpers.rb]: This is a good idea, we should probably look to export more to helper methods

File: app/models/space.rb [hunk 2]
```diff
@@ -7,4 +7,6 @@ class Space
   property :price, Float
   property :from_date, Date
   property :to_date, Date
+
+  belongs_to :user, :required => false
```
Comment (domvernon) [inline app/models/space.rb]: We should look to remove, `:required => false` at some point.

**Comment by Tagrand** [comment]
:shipit:

**Comment by domvernon** [comment]
:shipit:

**Pull request merged**